lib/tesla/middleware/json.ex

```diff
@@ -151,7 +156,15 @@ defmodule Tesla.Middleware.JSON do
 end
 
 defmodule Tesla.Middleware.DecodeJson do
-  @moduledoc false
+  @moduledoc """
+  Decodes response body as JSON.
+
+  Only decodes the body if the `Content-Type` header suggests
+  that the body is JSON.
+  """
+  @moduledoc since: "1.8.0"
```

Comment:
Next version.

Reply:
The funny part is that I do know people using the module already 😄